apps/system/js/notifications.js

```diff
@@ -611,6 +624,9 @@ var NotificationScreen = {
     });
 
     notification.classList.add('disappearing');
+    if (duration) {
+      notification.style.transitionDuration = duration + 'ms';
+    }
```

Comment:
Do we want to do a transitionDuration = '' in an else block here, to make sure? (or if we add that const as mentioned above, transitionDuration = CONST_VALUE + 'ms'? (and then remove from CSS to avoid confusion too)